config/config.go

```diff
@@ -706,6 +713,8 @@ var defaultConf = Config{
 			// FIXME: Find an appropriate default limit.
 			KeysLimit:         256,
 			TotalKeySizeLimit: 4 * 1024, // 4 KiB
+			SafeWindow:        3 * time.Second,
```

Comment:
Maybe shorter is enough. like 2 seconds, with 500ms clock drift.

Reply:
ok